.env.example

```diff
@@ -3,4 +3,6 @@ SENTRY_PROJECT=
 SENTRY_AUTH_TOKEN=
 SENTRY_DSN=
 DOMAIN_FORMAT=
-
+APP_BUILD_VERSION=1
```

Comment:
these are already being set in commands like `npm run start`. might as well set in env so that all commands will work